bodhi/client/__init__.py

```diff
@@ -374,10 +374,15 @@ def edit(user, password, url, **kwargs):
               type=click.Choice(['newpackage', 'security', 'bugfix', 'enhancement']))
 @click.option('--user', help='Updates submitted by a specific user')
 @click.option('--mine', is_flag=True, help='Show only your updates')
+@click.option('--rows', default=None,
```

Comment:
We could avoid duplicating these options ( in query and query_buildroot_overrides). You can check how the `@url_option` or `@new_edit_options` decorator works.